css/style.css

```diff
@@ -115,11 +261,234 @@ h1.heading {
+}
+
+.quality-card {
+  border: 1px solid #c1c7d0;
+  border-radius: 0.5rem;
+  background-color: #ebf0ee;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: center;
+}
+
+.quality-name {
+  font-size: 1.5rem;
+  font-weight: 600;
+  text-align: center;
+  margin: 1.5rem 0;
+}
+
+.icon-wrapper {
+  border-radius: 100%;
+  background-color: white;
+  padding: 1.125rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 0 8px 16px 0 rgba(54, 179, 127, 0.24);
+}
+
+.contact-section {
+  height: 60vh;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-image: url(/assets/contact-bg.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: bottom;
+}
+
+.contact {
+  width: 72rem;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: start;
+}
+
+.contact-info {
+  color: #172b4d;
+  font-size: 2.5rem;
+  font-weight: 700;
+  line-height: 3.25rem;
+  max-width: 34rem;
+  text-align: left;
+}
+
+.contact-form {
+  width: 100%;
+}
+
+.form-block {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+}
+
+input[type=text] {
+  width: 95%;
+  padding: 0.5rem;
+  border: 1px solid #c1c7d0;
+  border-radius: 0.25rem;
+  margin: 0.5rem 0;
+  min-height: 24px;
+}
+
+.input-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 0.5rem;
+  justify-content: space-between;
+}
+
+input[type=email] {
+  width: 95%;
+  padding: 0.5rem;
+  border: 1px solid #c1c7d0;
+  border-radius: 0.25rem;
+  margin: 0.5rem 0;
+  min-height: 24px;
+}
+
+textarea {
+  width: 95%;
+  padding: 0.5rem;
+  border: 1px solid #c1c7d0;
+  border-radius: 0.25rem;
+  margin: 0.5rem 0;
+  min-height: 24px;
+}
+
+button.see-project {
+  text-decoration: none;
+  background-color: #36b37f;
+  padding: 0.75rem;
+  font-size: 1.0625rem;
+  color: #fff;
+  border-radius: 0.25rem;
+  margin: 0.5rem 0;
+  border-style: none;
+  cursor: pointer;
+}
+
+::placeholder {
+  color: #6b778c;
+  font-size: 0.9rem;
+  font-family: 'Inter', sans-serif;
+}
+
+input:focus {
+  outline: none !important;
+  border-color: #60c095;
+}
+
+textarea:focus {
+  outline: none !important;
+  border-color: #60c095;
+}
+
+button.see-project:hover {
+  background-color: #36b37f;
+  box-shadow: 0 8px 16px 0 rgba(54, 179, 127, 0.24);
+}
+
+button.see-project:focus {
+  background-color: #008552;
+  outline: 1px solid #008552;
+}
+
+.footer-section {
+  margin: 2.75rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+}
+
+a.email-link {
+  text-decoration: none;
+  color: #42526e;
+  font-size: 1.25rem;
+  font-weight: 400;
+  line-height: 32px;
+  margin: 0.5rem 0;
+  cursor: pointer;
+}
+
 @media screen and (max-width: 768px) {
+  section {
```

Comment:
- [ ] Please make sure that your design is mobile-first. So, consider using `min-width` instead of `max-width` here. Also, please make sure that your design looks appropriate to the Figma design in both desktop and mobile designs after you change it.